### DISCUSSION TITLE
Buid-in like in classical discord

### DISCUSSION THREAD

**Discussion by VACRandomSample** [Rapptz/discord.py#7322] [Q&A]
Need claas which use to create hints
![image](https://user-images.githubusercontent.com/88278350/127773029-39cb8077-f5a8-4fad-a7eb-82ef8fad2e72.png)

**Comment by lgaan** [comment]
These are slash commands, which are not yet implemented in the latest or master version of the library.

**Reply by VACRandomSample** [reply]
but how that`s bots use it?

**Reply by lgaan** [reply]
Either different libraries, or they’ve implemented it themselves (latter option is more likely).

**Comment by VACRandomSample** [comment]


**Reply by mrvillage** [reply]
Don’t recommend third party libraries.
